src/parser/shared/modules/CooldownThroughputTracker.tsx

```diff
@@ -80,6 +80,14 @@ class CooldownThroughputTracker extends Analyzer {
       spell: TALENTS_PRIEST.POWER_INFUSION_TALENT.id,
       summary: [BUILT_IN_SUMMARY_TYPES.DAMAGE, BUILT_IN_SUMMARY_TYPES.HEALING],
     },
+    {
+      spell: SPELLS.HEROISM.id,
+      summary: [BUILT_IN_SUMMARY_TYPES.DAMAGE, BUILT_IN_SUMMARY_TYPES.HEALING],
+    },
+    {
+      spell: SPELLS.BLOODLUST.id,
+      summary: [BUILT_IN_SUMMARY_TYPES.DAMAGE, BUILT_IN_SUMMARY_TYPES.HEALING],
+    },
```

Comment:
I don't think this is a change that we want for retail. can you make it conditional on the expansion? (`this.owner.config.expansion` should tell you)